pkg/acsengine/template_generator.go

```diff
@@ -161,7 +161,7 @@ func (t *TemplateGenerator) getMasterCustomData(cs *api.ContainerService, textFi
 	}
 
 	// add manifests
-	str = substituteConfigString(str,
+	str = substituteAddonConfigString(str,
```

Comment:
After we remove `kubernetesAddonSetting` we can just go back to using `substituteConfigString`. So the code to use the rawScript directly should be in `substituteConfigString`.